back/app/finders/events_finder.rb

```diff
@@ -14,11 +14,11 @@ def project_publication_statuses_condition(project_publication_statuses)
     scope(:with_project_publication_statuses, project_publication_statuses) if project_publication_statuses.present?
   end
 
-  def start_at_lt_condition(start_at)
-    where('start_at < ?', start_at)
+  def end_at_lt_condition(end_at)
+    where('end_at < ?', end_at)
   end
 
-  def start_at_gteq_condition(start_at)
-    where('start_at >= ?', start_at)
+  def end_at_gteq_condition(end_at)
+    where('end_at >= ?', end_at)
```

Comment:
We now sort into past vs. not-past according to wether `end_date` is less than a given date (the date now) OR greater than or equal to a given date. Previously `start_date` was used, which caused ongoing events to be grouped with past events.